great_expectations_cloud/agent/actions/unknown.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from typing_extensions import override
+
+from great_expectations_cloud.agent.actions.agent_action import (
+    ActionResult,
+    AgentAction,
+)
+from great_expectations_cloud.agent.models import (
+    UnknownEvent,
+)
+
+
+class UnknownEventAction(AgentAction[UnknownEvent]):
+    @override
+    def run(self, event: UnknownEvent, id: str) -> ActionResult:
+        # noop
```

Comment:
I think we should emit a warning here.
https://docs.python.org/3/library/warnings.html

Reply:
Agree

Reply:
added warning here, then an error upon failed parse in 350b8b5

Reply:
This is not a real warning.
`LOGGER.warning` `!=` `warnings.warn()`

Reply:
oh oversight. looking at it now

Reply:
You'll likely also need to catch this warning in tests.
I would also suggest creating a custom `class GxAgentWarning(UserWarning):...`
https://docs.pytest.org/en/4.6.x/warnings.html#warns
```python
    with pytest.warns(GxAgentWarning):
        code_that_will_throw_this_warning()
```

Reply:
Check out [new user warning type](https://github.com/great-expectations/cloud/pull/154/files#diff-a89ee08302d15cae4b00da96e8840a01a643d2cb0a82d83aa606a1b300457c9eR6), [raise warning](https://github.com/great-expectations/cloud/pull/154/files#diff-a89ee08302d15cae4b00da96e8840a01a643d2cb0a82d83aa606a1b300457c9eR10), and  [test](https://github.com/great-expectations/cloud/pull/154/files#diff-654cf3c89d7235a8c47412339c29ae838d0e8fd9e78de7e08bc6e10b89c0ba83R12)